frontend/sozisel-app/src/components/SessionsList/SessionsList.tsx

```diff
@@ -0,0 +1,182 @@
+        setSuccessMessage(`${t("components.SessionsList.sessionDeleted")}`);
+      } catch (error) {
+        console.error(error);
+      }
+    },
+    [deleteMutation, refetch, filters, t]
+  );
+
+  const onApplyFilters = useCallback(
+    (filters: SessionFiltersSchema) => {
+      setSearchName("");
+      setFilters(filters);
+      refetch({ input: filters });
+    },
+    [refetch]
+  );
+
+  const onSearchNameChange = (event: BaseSyntheticEvent) => {
+    setSearchName(event.target.value);
+    debounceFetch(event.target.value);
+  };
+
+  const onSearchNameCleared = () => {
+    setSearchName("");
+    debounceFetch(undefined);
+  };
+
+  if (loading) {
+    return (
+      <>
+        <MainNavbar></MainNavbar>
+        <div className="SessionsListContainer">
+          <SessionFilters onSubmitCallback={onApplyFilters} />
+          <CircularProgress></CircularProgress>
+        </div>
+      </>
+    );
+  }
+
+  if (data?.searchSessions) {
+    return (
+      <>
+        <MainNavbar></MainNavbar>
+        <div className="SessionsListContainer">
+          <SessionFilters onSubmitCallback={onApplyFilters} />
+          <div className="listWithSearchBar">
+            <div className="searchBarContainer">
+              <TextField
+                fullWidth
+                id="searchTextInput"
+                variant="outlined"
+                size="small"
+                value={searchName}
+                onChange={onSearchNameChange}
+                InputProps={{
+                  startAdornment: (
+                    <InputAdornment position="end">
+                      <SearchIcon />
+                    </InputAdornment>
+                  ),
+                  endAdornment: (
+                    <InputAdornment position="end">
+                      {searchName != "" && (
+                        <ClearIcon
+                          color="primary"
+                          onClick={onSearchNameCleared}
+                          cursor="pointer"
+                        />
+                      )}
+                    </InputAdornment>
+                  ),
+                }}
+              />
+            </div>
+            <List>
+              {data.searchSessions.map((element, _) => (
+                <SessionCard
+                  key={element.id}
+                  session={element}
+                  onDelete={onSessionDelete}
+                />
+              ))}
+            </List>
+          </div>
+        </div>
+        <Snackbar
+          open={successMessage !== ""}
+          autoHideDuration={6000}
+          onClose={() => setSuccessMessage("")}
+        >
+          <Alert onClose={() => setSuccessMessage("")} severity="success">
+            {successMessage}
+          </Alert>
+        </Snackbar>
+        <Snackbar open={deleteLoading} autoHideDuration={3000}>
+          <CircularProgress />
+        </Snackbar>
+        <Snackbar open={!!deleteError} autoHideDuration={6000}>
```

Comment:
Why once you have 6 seconds and the other time 3 seconds? What is the difference?

Reply:
I will change it to 6 seconds.